<?xml version="1.0" ?>
<launch>
    <include file="$(find franka_hw)/launch/franka_hw.launch" />
    <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="cartesian_pose_example_controller --shutdown-timeout 2"/>
    <remap from="joint_states" to="franka_hw_node/joint_states"/>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" output="screen"/>
    <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_description)/rviz/franka_description.rviz"/>
</launch>
